spring-rest-http/src/main/java/com/baeldung/jsonparam/jsonParamOpenApiSwaggerDefinition

```diff
@@ -0,0 +1,154 @@
+swagger: '2.0'
+...
+paths:
+  /tickets:
+    get:
+      parameters:
+        - in: query
+          name: params
+          required: true
+          description: A JSON object with the `type` and `color` properties
+          type: string
+          example: '{"type":"foo","color":"green"}'
+
+swagger: '2.0'
+...
+paths:
+  /tickets:
+    post:
+      requestBody:
+          description: Parameter is an object that should be serialized as JSON
+          content:
+            application/json:
+              schema:
+              	type: string
+          		example: '{"type":"foo","color":"green"}'
+      responses:
+        '200':
+          description: successful process
+
+"/api/tickets": {
```

Comment:
Two things starting with this line: First, the indentation is off. This path and the paths below should be indented the same as /tickets above. Second, why do we start putting everything in double quotes at this point? Although it is allowed, convention is to not use any quotes in an OpenAPI spec except where necessary.